.github/workflows/pull-request.yaml

```diff
@@ -1,6 +1,9 @@
 name: Pull request
 on:
 - pull_request
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
added this since i noticed that runs from old commits were still going. any reason not to do this?

Reply:
sure, why not